docs/source/tutorials.rst

```diff
@@ -6,3 +6,4 @@ Tutorials
 
    tutorials/publication_quality_figures.rst
    tutorials/composing_multipanel_figures.rst
+   tutorials/getting_started.rst
```

Comment:
it will be probably better to put it in the `index.rst` just before the line with `tutorials`